src/frontend/src/components/WebformTable/components/Row/index.tsx

```diff
@@ -167,9 +169,11 @@ export default React.memo(function Row({
 
   return (
     <StyledTableRow component="div">
-      <StyledTableCell component="div">
-        <Id>{id}</Id>
-      </StyledTableCell>
+      {!skidIdColumn && (
```

Comment:
is this meant to be `skipIdColumn` instead of skid?
Prefer for a boolean prefix, like `shouldSkipIdColumn` or something.

Reply:
yes it was supposed to be `skipIdColumn`, i'll rename to `shouldSkipIdColumn`